Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandTransmuteItems.java

```diff
@@ -47,6 +47,10 @@ public void run(Map<String, String> parameters) throws UserErrorMessage {
 		
 		Long containerId = tryParseId(parameters, "containerId");
 		final Key containerKey = KeyFactory.createKey("Item", containerId);
+		CachedEntity container = db.getEntity(containerKey);
+		
+		if (!(boolean) container.getProperty("transmuteEnabled"))
```

Comment:
This will cause a NPE for pretty much all "not transmute box" cases. Better to use GameUtils.equals(..getProperty(), true) imo